packages/web-components/fast-components/src/number-field/number-field.definition.ts

```diff
@@ -0,0 +1,126 @@
+                    required: false,
+                    type: DataType.string,
+                    default: undefined,
+                },
+                {
+                    name: "list",
+                    description: "The list attribute",
+                    required: false,
+                    type: DataType.string,
+                    default: "",
+                },
+                {
+                    name: "maxlength",
+                    description: "The maxlength attribute",
+                    required: false,
+                    type: DataType.number,
+                    default: undefined,
+                },
+                {
+                    name: "minlength",
+                    description: "The minlength attribute",
+                    required: false,
+                    type: DataType.number,
+                    default: undefined,
+                },
+                {
+                    name: "size",
+                    description: "The size attribute",
+                    required: false,
+                    type: DataType.number,
+                    default: undefined,
+                },
+                {
+                    name: "name",
+                    description: "The name attribute",
+                    type: DataType.string,
+                    default: "",
+                    required: false,
+                },
+                {
+                    name: "required",
+                    description: "The required attribute",
+                    type: DataType.boolean,
+                    default: false,
+                    required: false,
+                },
+                {
+                    name: "disabled",
+                    description: "The disabled attribute",
+                    type: DataType.boolean,
+                    default: false,
+                    required: false,
+                },
+                {
+                    name: "readonly",
+                    description: "The readonly attribute",
+                    type: DataType.boolean,
+                    default: false,
+                    required: false,
+                },
+                {
+                    name: "max",
+                    description: "The maximum value",
+                    type: DataType.string,
+                    default: false,
+                    required: false,
+                },
+                {
+                    name: "min",
+                    description: "The minimum value",
+                    type: DataType.string,
+                    default: false,
+                    required: false,
+                },
+                {
+                    name: "step",
+                    description: "The step interval",
+                    type: DataType.string,
+                    default: false,
+                    required: false,
+                },
+            ],
+            slots: [
```

Comment:
There are start and end slots as well yes?

Reply:
Added start and end slots.